src/Tribe/Views/V2/Messages.php

```diff
@@ -94,6 +94,10 @@ public static function for_key( $key, ...$values ) {
 				'There were no results found.',
 				'the-events-calendar'
 			),
+			'no_upcoming_events'               => __(
```

Comment:
Looks good, can you use the function for the word events here? ( plural lowercase )